main.go

```diff
@@ -339,6 +355,11 @@ func commonMain() (*FileConfig, http.Handler) {
 		logger.Error("Error loading/parsing config file", "err", err)
 		os.Exit(2)
 	}
+	emailsConfig, err := loadEmailsConfig(*emailsCfg)
+	if err != nil {
+		logger.Error("Error loading/parsing emails config file", "err", err)
```

Comment:
And here

Reply:
The other thing (if we are breaking backward compatibility) would be to give a hint to people that they need to add this file if they recently upgraded.